nav2_msgs/msg/PolygonsArray.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
```

Comment:
Since each `PolygonStamped` in the array below has its own header, I think, main header is unnecessary